### DISCUSSION TITLE
Principle: A proposed standard must not compete with other FINOS standards

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#17] [General]
Why? Giving members the choice between different projects that serve the same use case, or implement the same standards, is considered a good thing. An element of competition generally results in an ever-evolving and improving ecosystem of solutions. However, competing standards have the opposite effect, they result in a fractured ecosystem where competition results in exclusion.

**Comment by rikoe** [comment]
Agreed. The tricky thing here is defining "competes with".

**Reply by ColinEberhardt** [reply]
Rather than attempt to define this term (which I don't think is possible!), I'd prefer to leave it to the judgement of the TSC.

**Comment by ColinEberhardt** [comment]
via our TSC list - "Compete or overlap with?"
